### PR TITLE
Edit URL deployment to GitPod

### PR DESCRIPTION
**Pull request by thomasbnt**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [x] Documentation Update

## Description
Just edited the URL to deploy on **GitPod**.

```diff
- <a href="https://gitpod.io/from-referrer/">
+ <a href="https://gitpod.io/#https://github.com/forem/forem">
	<img src="https://img.shields.io/badge/setup-automated-blue?logo=gitpod" alt="GitPod badge">
</a>
```

image

## Related Tickets & Documents

Nope.

## QA Instructions, Screenshots, Recordings

_Please replace this line with instructions on how to test your changes, a note
on the devices and browsers this has been tested on, as well as any relevant
images for UI changes._

### UI accessibility concerns?

_If your PR includes UI changes, please replace this line with details on how
accessibility is impacted and tested. For more info, check out the
[Forem Accessibility Docs](https://developers.forem.com/frontend/accessibility)._

## Added/updated tests?

- [ ] Yes
- [x] No, and this is why: Just an updated line with the true URL.
- [ ] I need help with writing tests

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

- [ ] I've updated the [Developer Docs](https://developers.forem.com) or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
- [ ] This PR changes the Forem platform and our documentation needs to be
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
- [x] I've updated the README or added inline documentation
- [ ] I've added an entry to
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
- [ ] I will share this change in a [Changelog](https://forem.dev/t/changelog)
      or in a [forem.dev](http://forem.dev) post
- [ ] I will share this change internally with the appropriate teams
- [ ] I'm not sure how best to communicate this change and need help
- [ ] This change does not need to be communicated, and this is why not: _please
      replace this line with details on why this change doesn't need to be
      shared_

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@
   <a href="https://github.com/forem/forem/issues?q=is%3Aissue+is%3Aopen+label%3A%22ready+for+dev%22">
     <img src="https://img.shields.io/github/issues/forem/forem/ready for dev" alt="GitHub issues ready for dev">
   </a>
-  <a href="https://gitpod.io/from-referrer/">
+  <a href="https://gitpod.io/#https://github.com/forem/forem">
```
Comment (nickytonline) [inline README.md:15]: This is the correct URL for launching Gitpod, but only if you are a core contributor or want a version of Forem to look at and run.

I'm wondering if providing the format of the URL would be more helpful to potential contributors, e.g.

```markdown
To launch Forem in Gitpod, navigate to https://gitpod.io/#https://github.com/your_github_username/forem
```

Thoughts @amywtlin ?

Comment (nickytonline) [reply README.md:15]: Sorry, I meant the new URL you added is correct, https://gitpod.io/#https://github.com/forem/forem, but only for core contributors.

Comment (thomasbnt) [reply README.md:15]: I've got only https://github.com in referral headers. :c
image

Comment (amywtlin) [reply README.md:15]: One way that might help is to stress `your_github_username` with brackets surrounding it so it is more obvious that whoever copying & pasting the URL needs to replace that parameter with their own username for it to work. 

From: 
```To launch Forem in Gitpod, navigate to https://gitpod.io/#https://github.com/your_github_username/forem```
To: 
```To launch Forem in Gitpod, navigate to https://gitpod.io/#https://github.com/{your_github_username}/forem```

I am not sure if we have used one specific type of brackets in this case... if we have, we should stick to the same type. cc @aitchiss as I feel like you would know if we have opted for a certain type of brackets implying it's a parameter.

Comment (aitchiss) [reply README.md:15]: Yeah I would suggest the same as you @amywtlin, with the `{}` brackets 👍

**Comment by msarit** [comment]
@thomasbnt Hey hey! Please update your code according to the suggestion [here](https://github.com/forem/forem/pull/16795/files#r821266235), and we'll proceed to merge this PR! Thank you for your controbution!

**Comment by thomasbnt** [comment]
> @thomasbnt Hey hey! Please update your code according to the suggestion [here](https://github.com/forem/forem/pull/16795/files#r821266235), and we'll proceed to merge this PR! Thank you for your controbution!


Hello @msarit ! I added text in README about **How to launch Forem with Gitpod**. (https://github.com/forem/forem/pull/16795/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R82-R83)

**Review by jeremyf — APPROVED** [review]

**Review by maestromac — APPROVED** [review]

**Pull request merged**